src/main/java/org/jenkinsci/test/acceptance/po/UpdateCenter.java

```diff
@@ -71,7 +70,8 @@ public boolean isInstalled(String pluginShortName) throws InstallationFailedExce
      */
     public boolean waitForInstallationToComplete(final PluginSpec... specs) throws InstallationFailedException {
         open();
-        waitFor(driver, not(Matchers.hasContent("Pending")), 60); // Wait for all plugins to get installed
+        // Wait for all plugins to get installed
+        waitFor(driver, not(Matchers.hasElement(by.xpath("//*[@id='log']//*[contains(.,'Pending') or contains(.,'Installing')]"))), 60);
```

Comment:
@vilacides checking only for 'Pending' is not sufficient, and to check for the 'Installing' string we need to be more precise as explained just above, hence the xpath.